docs/docs-beta/sidebars.ts

```diff
@@ -11,7 +11,11 @@ const sidebars: SidebarsConfig = {
       type: 'category',
       label: 'Tutorial',
       collapsed: false,
-      items: ['tutorial/tutorial-etl'],
+      items: [
+        'tutorial/etl-tutorial-introduction',
+        'tutorial/create-and-materialize-assets',
+        'tutorial/creating-a-downstream-asset',
```

Comment:
As mentioned earlier, I would call this "create and materialize a downstream asset" for consistency